de.dlr.sc.virsat.project/src/de/dlr/sc/virsat/project/editingDomain/VirSatWorkspaceCommandStack.java

```diff
@@ -113,10 +114,23 @@ public void executeNoUndo(Command command) {
 					operation.execute(null, null);
 				} catch (ExecutionException e) {
 					Activator.getDefault().getLog().log(new Status(Status.ERROR, Activator.getPluginId(), "VirSatWorkspaceCommandStack: Failed to execute command without undo ", e));
+					atomicExecutionException.set(e);
 				}
 			}
-		});
+			
+			// Now run the builder if requested
+			if ((executeBuilder) && !ResourcesPlugin.getWorkspace().isAutoBuilding()) {

```

Comment:
Does it make sense to execute the builder if the command was not executeable or there was an exception?
I think this might make it harder to reproduce issues, say, if there is a problem with some command since the builder will change the state of the model.

Reply:
I wasn't realy sure how to deal with it, but actually thought about it. Since i don't have a strong opinion on either the one or the other, I am happy to follow.